pkg/controller/configmap/config.go

```diff
@@ -431,3 +453,29 @@ func optionInConfigMap(configMap *corev1.ConfigMap, section string, key string)
 	}
 	return false
 }
+
+func getOptionInConfigMap(configMap *corev1.ConfigMap, section string, key string) string {
+	cfg, err := ini.Load([]byte(configMap.Data[operatortypes.ConfigMapDataKey]))
+	if err != nil {
+		log.Error(err, "Failed to load ConfigMap")
+		return ""
+	}
+	sec, err := cfg.GetSection(section)
+	if err != nil {
+		return ""
+	}
+	if sec.HasKey(key) {
+		return sec.Key(key).Value()
+	}
+	return ""
+}
+
+func IsMTUChanged(currConfigMap *corev1.ConfigMap, prevConfigMap *corev1.ConfigMap) bool {
```

Comment:
not necessary but, do you think we can be proactive and validate new MTU size before updating configmap and restarting pods?

Reply:
Good suggestion, we can add MTU validation in operator